clang/lib/Sema/SemaStmtAttr.cpp

```diff
@@ -334,11 +334,12 @@ CodeAlignAttr *Sema::BuildCodeAlignAttr(const AttributeCommonInfo &CI,
     // This attribute requires an integer argument which is a constant power of
     // two between 1 and 4096 inclusive.
     int AlignValue = ArgVal.getSExtValue();
-    if (AlignValue < CodeAlignAttr::getMinValue() ||
-        AlignValue > CodeAlignAttr::getMaxValue() || !ArgVal.isPowerOf2()) {
+    static int MaximumAlignment = 4096;
```

Comment:
Ah,  I meant these still as addition members, as static-constexpr.  That way it is still `CodeAlignAttr::MinimumAlignment` and `CodeAlignAttr::MaximumAlignment`, but not as a function that implies that it is not a constant value.

Reply:
I see! Thanks @erichkeane for clarification. Fixed now with commit https://github.com/llvm/llvm-project/pull/70762/commits/aa85ed1ede892c7da3d5c2dee5886dd0e2ad0cb7